partner_email_check/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========
+Email check
+===========
+
+This module validate the field `email` in the module `res.partner`.
```

Comment:
This modules validates the format email field of the Partner model (res.partner) using the python package validate_email.

Reply:
This is still missing, and remember to use double backticks for RST when you want to represent code.